ksql-common/src/main/java/io/confluent/ksql/util/KsqlConfig.java

```diff
@@ -201,6 +201,18 @@
       "Config to enable or disable transient pull queries on a specific KSQL server.";
   public static final boolean KSQL_PULL_QUERIES_ENABLE_DEFAULT = true;
 
+  public static final String KSQL_PULL_QUERIES_OWNER_TIMEOUT_MS_CONFIG =
+      "ksql.pull.queries.owner.timeout.ms";
```

Comment:
To follow the convention that `KSQL_ACTIVE_PERSISTENT_QUERY_LIMIT_CONFIG` follows
`ksql.query.persistent.active.limit`
```suggestion
      "ksql.query.pull.owner.timeout.ms";
```

Reply:
Never mind, what you have follows the `KSQL_PULL_QUERIES_ENABLE_CONFIG` naming convention so it should be fine.

Reply:
@agavra how would you feel about renaming the pull query enable config to 
`ksql.query.pull.enable` to keep all these query configs consistently named

Reply:
sounds good to me - just make sure we update our cloud config to reflect the new value

Reply:
some context for @AlanConfluent - this config hasn't been released yet so we can change it without worrying about compatibility

Reply:
Great, I changed it and made it consistent.